### ISSUE TITLE
Does this work on Linux which is no twincat?

### ISSUE THREAD

**Issue by jianingli1997** [aslze/beckhoffAds#1]
Hi ,Does this work on Linux which is no twincat?

**Comment by aslze** [comment]
Yes, this should be cross-platform in principle (Windows, Linux, Mac, Android). I think I tried Linux, not sure. The library just does standard TCP/IP networking.

Of course there must be a TwinCAT machine (Windows, I guess) with which the Linux machine communicates on the same network. And the TwinCAT system should have the necessary *TwinCAT Routes* set-up to allow the remote machine to connect.

**Comment by aslze** [comment]
This is answered. Yes it works on Linux (on a machine separate from the Twincat/PLC one in the same network).
